specification_documents/examples/intro_qc2.mzQC

```diff
@@ -142,8 +174,8 @@
     "controlledVocabularies": [
       {
         "name": "Proteomics Standards Initiative Mass Spectrometry Ontology",
-        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.79/psi-ms.obo",
-        "version": "4.1.79"
+        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.157/psi-ms.obo",
+        "version": "4.1.157"
```

Comment:
**todo:** Version number needs to be updated after [psi-ms-CV#318](https://github.com/HUPO-PSI/psi-ms-CV/pull/318) has been merged.